### PR TITLE
op-guide: update tidb-ansible deployment

### PR DESCRIPTION
**Pull request by liubo0127**
@lilin90 @superlzs0476 PTAL

### CODE REVIEW

File: op-guide/ansible-deployment.md [hunk 1]
```diff
@@ -115,29 +115,24 @@ The key's randomart image is:
...
-| master | master 版本 | 包含最新特性，每日更新。 |
+| TiDB 版本 | tidb-ansible tag | 备注 |
+| -------- | ---------------- | --- |
+| 2.0 版本 | v2.0.10、v2.0.11 | 最新 2.0 稳定版本，可用于生产环境。 |
+| 2.1 版本 | v2.1.1 ~ v2.1.6 | 最新 2.1 稳定版本，可用于生产环境（建议）。 |
+| 3.0 版本 | v3.0.0-beta、v3.0.0-beta.1 | 目前是 beta 版本，不建议用于生产环境。 |
+| latest 版本 | None | 包含最新特性，每日更新，不建议用于生产环境。 |
 
-使用以下命令从 Github [TiDB-Ansible 项目](https://github.com/pingcap/tidb-ansible)上下载 TiDB-Ansible 相应分支，默认的文件夹名称为 `tidb-ansible`。
+使用以下命令从 Github [TiDB-Ansible 项目](https://github.com/pingcap/tidb-ansible)上下载 TiDB-Ansible 相应版本，默认的文件夹名称为 `tidb-ansible`。注意：部署和升级 TiDB 集群要使用对应的 tidb-ansible，通过改 inventory.ini 中的版本来混用可能会产生一些错误。
```
Comment (lilin90) [inline op-guide/ansible-deployment.md]:
```suggestion
使用以下命令从 Github [TiDB-Ansible 项目](https://github.com/pingcap/tidb-ansible)上下载 TiDB-Ansible 相应版本，默认的文件夹名称为 `tidb-ansible`。

> **注意**：部署和升级 TiDB 集群要使用对应的 tidb-ansible 版本，通过改 `inventory.ini` 文件中的版本来混用可能会产生一些错误。
```

Comment (lilin90) [reply op-guide/ansible-deployment.md]: How to understand "来混用" here?

File: op-guide/offline-ansible-deployment.md [hunk 2]
```diff
@@ -80,21 +89,19 @@ category: deployment
 
 2.  下载 tidb-ansible
 
-    使用以下命令从 Github [TiDB-Ansible 项目](https://github.com/pingcap/tidb-ansible)上下载 TiDB-Ansible 相应版本，默认的文件夹名称为 `tidb-ansible`，以下为各版本下载示例，版本选择可以咨询官方。
+    使用以下命令从 Github [TiDB-Ansible 项目](https://github.com/pingcap/tidb-ansible)上下载 TiDB-Ansible 相应版本，默认的文件夹名称为 `tidb-ansible`。注意：部署和升级 TiDB 集群要使用对应的 tidb-ansible，通过改 inventory.ini 中的版本来混用可能会产生一些错误。
```
Comment (lilin90) [inline op-guide/offline-ansible-deployment.md]:
```suggestion
    使用以下命令从 Github [TiDB-Ansible 项目](https://github.com/pingcap/tidb-ansible)上下载 TiDB-Ansible 相应版本，默认的文件夹名称为 `tidb-ansible`。
    
        > **注意**：部署和升级 TiDB 集群要使用对应的 tidb-ansible 版本，通过改 `inventory.ini` 文件中的版本来混用可能会产生一些错误。
```

File: op-guide/ansible-deployment.md (hunk 1 above)
Comment (liubo0127) [reply op-guide/ansible-deployment.md]: This means deploying TiDB using `tidb-ansible` with a mismatched version, such as `tidb-ansible` with v2.1.4 tag to deploy v2.1.6 version TiDB.

File: op-guide/offline-ansible-deployment.md (hunk 2 above)
Comment (liubo0127) [reply op-guide/offline-ansible-deployment.md]: May have more spaces before `> **注意**：`

Comment (lilin90) [reply op-guide/offline-ansible-deployment.md]: @liubo0127 How about now?

File: op-guide/ansible-deployment.md (hunk 1 above)
Comment (lilin90) [reply op-guide/ansible-deployment.md]: Got it. Thanks!

File: op-guide/offline-ansible-deployment.md (hunk 2 above)
Comment (liubo0127) [reply op-guide/offline-ansible-deployment.md]: I think it's okay.

**Pull request merged**